<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Luckenbach RNA</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Luckenbach RNA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_take_liver">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Take Liver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Current Barcode</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_cur_code">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_nad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Gonad RNA Taken</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_liver">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Liver RNA Taken</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_done">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_samp_num">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Use Sample Number: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_take_gonad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Take Gonad</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
